app/assets/javascripts/api_stub.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Created by kaceyryan on 5/2/15.
+ */
+
+var request = require('request');
+
+class API {
+  //get images in JSON object for your location
+  getImages(callback) {
```

Comment:
All of these functions should accept a `callback` and an `errorCallback` argument, in addition to the regular ones like `id`, `image`, etc.